src/tests/ftest/util/AgentUtils.py

```diff
@@ -118,4 +171,4 @@ def stop_agent(client_list):
     # this kills the agent
     for client in client_list:
         cmd = "pkill daos_agent"
-        resp = subprocess.call(["ssh", client, cmd])
\ No newline at end of file
+        subprocess.call(["ssh", client, cmd])
```

Comment:
What happens if this ssh fails and the agent is not killed (perhaps for a different reason even)? Certainly for the case of where it's just being stopped during end of test cleanup, the agent not stopping is just annoying, but this function could be called in the middle of a test run where the agent failing to stop to could follow-on failures.

A failure to stop the agent should be an error of some sort.

Reply:
@sdwillso Will have to comment on this further but we looked at your comment from last time and there really isn't anything we can do here. This is the same thing that happens when you try to kill the server.

Reply:
We went with simple pkill the same way that ServerUtils does, seemed reasonable that if we've been ok with pkill for server kill, we'd be alright with it for agent kill.

Reply:
The server stop process first tries to stop the server gracefully through `orterun`.  It then goes and checks if any servers are still alive and if there are, it kills them directly and raises an exception so that the test where the servers fail to stop fails.

Reply:
Added in a check mirroring what ServerUtils does in stop_server(). We now kill via pkill and just double check to make sure it died, and if not, raise an exception so user is aware something is amiss.